action.yaml

```
@@ -59,6 +59,8 @@ inputs:
   st2_packs:
     description: String. Comma separated list of packs to install. This flag does not work with a --python3 only pack. (default `"st2"`). If you modify this option, be sure to also include `st2` in the list.
     default: "st2"
+  st2_conf:
+    description: Relative path from Project Root to configuration file. This deletes [the default st2.yaml](operations/deployment/ansible/vars/st2.yaml) file.
```

It's not just about configuring the `st2.conf`, but any configuration that's available in the https://github.com/stackStorm/ansible-st2#variables, so the naming might be something like this (to give an idea): 
```suggestion
  st2_ansible_vars_file:
    description: Relative path from Project Root to Ansible vars file. If you'd like to more detailed configuration like st2 version, st2.conf, RBAC, chatops, auth. See https://github.com/stackStorm/ansible-st2#variables for possibilities.
```

If we'd allow passing this file from repo to ansible-st2 roles, - that'll cover many advanced (2nd day) configuration cases, not just st2.conf.

---

Oh, don't forget to adjust the `README.md` too as that mirrors the `action.yaml` descriptions.